### ISSUE TITLE
Alt-drag to copy should persist z-index of copied element

### ISSUE THREAD

**Issue by xixixao** [excalidraw/excalidraw#2147]
It feels that if I'm copying an element (especially with alt-drag), I'd want it to stay at the same "height" (z-index, order), as the copied element.

In my case I have an arrow and a label on top of it and I'm adding more by copy pasting, but the new arrows always jump to the front.

**Comment by dwelle** [comment]
duplicate of https://github.com/excalidraw/excalidraw/issues/2073 (blocked by https://github.com/excalidraw/excalidraw/pull/2065)
